.gitignore

```diff
@@ -39,4 +39,6 @@ keys.json
 .tsbuildinfo
 
 # temporary remove development icon
-assets/icon.development.png
\ No newline at end of file
+assets/icon.development.png
+
+notes.txt
```

Comment:
Don't think this is necessary, unless there's reason

Reply:
Updated .gitignore to remove notes.txt